sklearn/datasets/tests/test_openml.py

```diff
@@ -1108,12 +1108,12 @@ def test_fetch_openml_emotions(monkeypatch, gzip_response):
     data_name = "emotions"
     data_version = 3
     target_column = [
-        "amazed.suprised",
+        "amazed.surprised",
```

Comment:
This typo should not be fixed as it's part of the original dataset.

```suggestion
        "amazed.suprised",
```

Reply:
Ouch. I don't know why these were still present. Fixed now.